presto-main/src/main/java/com/facebook/presto/sql/planner/LocalExecutionPlanner.java

```diff
@@ -930,7 +930,9 @@ public PhysicalOperation visitWindow(WindowNode node, LocalExecutionPlanContext
                     sortOrder,
                     node.getPreSortedOrderPrefix(),
                     10_000,
-                    pagesIndexFactory);
+                    pagesIndexFactory,
+                    isSpillEnabled(session),
```

Comment:
Please introduce additional temporary feature toggle `experimental.spill-window-functions` and corresponding session property `spill_window_functions` which must _additionally_ be enabled to enable this functionality. The rationale is that this, as any other new implementation, may have issues. Without separate toggle, should any issues arise, users will need to disable spill altogether.